backend/routes/products.js

```diff
@@ -34,16 +34,29 @@ router.get('/category/:id', async (req, res, next) => {
 
 router.post('/', async (req, res, next) => {
   try {
-    const newProduct = await Product.create(req.body);
+    const newProduct = await Product.create({
+      name: req.body.name,
+      imageUrl: req.body.imageUrl,
+      price: req.body.price,
+      brand: req.body.brand,
+      quantity: req.body.quantity,
+      description: req.body.description,
+    });
```

Comment:
consider protecting certain routes for admins only .  this pattern for middleware may be useful for certain routes: ('/', isAdmin, () =>{})